src/ert/run_models/ensemble_smoother.py

```diff
@@ -85,6 +85,10 @@ def run_experiment(
                 parameters_config, design_matrix_group = (
                     design_matrix.merge_with_existing_parameters(parameters_config)
                 )
+                if not any(p.update for p in parameters_config):
+                    raise ConfigValidationError(
+                        "Design matrix parameters cannot be updated!"
```

Comment:
If you have the extra context:
```suggestion
                        f"No parameters to update, all parameters are set to update:false, parameters from DESIGN_MATRIX: {list_of_parameter_keys}"
```

If not a simpler reword:
```suggestion
                        f"No parameters to update, all parameters are set to update:false"
```